scripts/lb-wrapper

```diff
@@ -40,21 +40,30 @@ fi
 #     exit 1
 # fi
 
```

Comment:
Validation on Lines 43-44 should definitely be restored in my opinion:

```
if ! command -v "${XKLB_EXECUTABLE}"; then
    log "Error" "xklb could not be found. Please install xklb and try again."
```

(...followed by a blank line if possible!)

Reply:
Done